neurodamus/ngv.py

```diff
@@ -191,22 +234,27 @@ def _show_mcd(sec):
     #        )
 
     def set_pointers(self):
-        glut_list = self._glut_list
         c = self._cellref
```

Comment:
since lots is being changed, can we also change this one letter variable, or drop it all together and just use `self._cellref`?

Reply:
yes, np with that. Done